doc/introduction/configuration.rst

```diff
@@ -33,15 +33,29 @@ and has the following format:
     authentication_token = "071cdcce-9241-4965-93af-4a4dbc739135"
     hostname = "localhost"
     use_ssl = true
+    port = 443
 
 Summary of options
 ------------------
 
-SF_API_USE_SSL:
-    Whether to use SSL or not when connecting to the API. True or False.
-SF_API_HOSTNAME:
-    The hostname of the server to connect to. Defaults to localhost. Must be one of the allowed
-    hosts.
-SF_API_AUTHENTICATION_TOKEN:
-    The authentication token to use when connecting to the API. Will be sent with every request in
-    the header.
+Keyword arguments
+*****************
+
+    **authentication_token (str)** (*required*)
+        The authentication token to use when connecting to the API. Will be sent with every request in
+        the header.
+    **hostname (str)** (*optional*)
+        The hostname of the server to connect to. Defaults to localhost. Must be one of the allowed
+        hosts.
+    **use_ssl (bool)** (*optional*)
+        Whether to use SSL or not when connecting to the API. True or False.
+    **port (int)** (*optional*)
+        The port to be used when connecting to the remote service.
+
+Environment variables
+*********************
+
+* SF_API_AUTHENTICATION_TOKEN
+* SF_API_HOSTNAME
+* SF_API_USE_SSL
+* SF_API_PORT
```

Comment:
I would combine the two sections above into one, something like (feel free to modify/clean):

```rest
Configuration options
*********************

 **authentication_token (str)** (*required*)
    The authentication token to use when connecting to the API. Will be sent with every request in
    the header. Corresponding environment variable: ``SF_API_AUTHENTICATION_TOKEN``

**hostname (str)** (*optional*)
    The hostname of the server to connect to. Defaults to ``localhost``. Must be one of the allowed
    hosts. Corresponding environment variable: ``SF_API_HOSTNAME``

**use_ssl (bool)** (*optional*)
    Whether to use SSL or not when connecting to the API. True or False.
    Corresponding environment variable: ``SF_API_USE_SSL``

**port (int)** (*optional*)
    The port to be used when connecting to the remote service.
    Corresponding environment variable: ``SF_API_PORT``
```

Reply:
Oh thanks for that!